lib/sections/home.dart

```diff
@@ -1,15 +1,333 @@
 import 'package:flutter/material.dart';
 
+// HOME PAGE
+// Home Page widget, stateless and holds country tags
 class HomePage extends StatelessWidget {
+  // Return container with list of countries
   Widget build(BuildContext context) {
     return Container(
-        padding: const EdgeInsets.symmetric(horizontal: 8.0, vertical: 200.0),
         child: Center(
-            child: Column(
-          children: [
-            Text("This is the Home Page", textScaleFactor: 2),
-            Icon(Icons.home, size: 150, color: Colors.blue),
+            child: ListView(
+      children: [
+        CountryTag("Bhutan", 'assets/flags/Bhutan.png'),
+        CountryTag("Bangladesh", 'assets/flags/Bangladesh.png'),
+        CountryTag("Afghanistan", 'assets/flags/Afghanistan.jpg'),
+      ],
+    )));
+  }
+}
+
+// Tag with name and flag of country
+// Wanted stateless, couldn't really get navigator to work with that, unsure why
```

Comment:
What sorts of errors are being thrown? Is there a particular reason you want stateless here? [This](https://flutter.dev/docs/development/ui/interactive#self-managed) may be helpful for you. I'll take a closer look into this